src/Features/CSharp/Portable/RemoveUnusedMembers/CSharpRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.RemoveUnusedMembers;
+
+namespace Microsoft.CodeAnalysis.CSharp.RemoveUnusedMembers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal class CSharpRemoveUnusedMembersDiagnosticAnalyzer
+        : AbstractRemoveUnusedMembersDiagnosticAnalyzer<DocumentationCommentTriviaSyntax, IdentifierNameSyntax>
+    {
+        public CSharpRemoveUnusedMembersDiagnosticAnalyzer()
+            : base(forceEnableRules: false)
+        {
+        }
+
+        // For testing purposes only.
```

Comment:
❕ Make sure we have an issue filed to have the test suite always enable rules for testing and remove this constructor.

Reply:
https://github.com/dotnet/roslyn/issues/29615